cypher/cypher-docs/src/docs/dev/query-tuning.asciidoc

```diff
@@ -63,7 +63,8 @@ Here we detail the available versions:
 |===
 | Query option | Description | Default
 | `3.5` | This will force the query to use Neo4j Cypher 3.5. |
-| `4.0` | This will force the query to use Neo4j Cypher 4.0. As this is the default version, it is not necessary to use this option explicitly. | X
+| `4.1` | This will force the query to use Neo4j Cypher 4.1. |
+| `4.2` | This will force the query to use Neo4j Cypher 4.2. As this is the default version, it is not necessary to use this option explicitly. | X
```

Comment:
What happened to the 4.0 version? Is it no longer available?

Reply:
It is not available in 4.2 any longer, that is right.